### PR TITLE
Add JAVA-API each versions of the document

### LINKED ISSUE #9653: Add  JAVA-API each versions of the document
Hi，all communities

Could you do me a small favor? We need to do a summary of the historical version of the document，So I created this task.

you  can see this : 

 [chang-history-cn](https://github.com/apache/shardingsphere/blob/master/docs/document/content/user-manual/shardingsphere-jdbc/configuration/java-api/change-history.cn.md)

[chang-history-en](https://github.com/apache/shardingsphere/blob/master/docs/document/content/user-manual/shardingsphere-jdbc/configuration/java-api/change-history.cn.md)

now, i will add 5.0.0-alpha version , you will add other version to this file

- [ ] 4.x document  
- [ ] 3.x document
- [ ] 2.x document
- [ ] 3.x document

Format is as follows : 

```
## 4.xxx （this is version）

### feture (like sharding, replica-query)

#### API config

```

Won't the PR be submitted yet?  you can see this : [how to contributor](https://shardingsphere.apache.org/community/en/contribute/contributor/)

### PR DESCRIPTION
**Pull request by xiaoshen11**
Fixes #9653.

Changes proposed in this pull request:
- docs\document\content\user-manual\shardingsphere-jdbc\configuration\java-api\change-history.cn.md
- docs\document\content\user-manual\shardingsphere-jdbc\configuration\java-api\change-history.en.md

### CODE REVIEW

File: docs/document/content/user-manual/shardingsphere-jdbc/configuration/java-api/change-history.en.md [hunk 1]
```diff
@@ -31,4 +31,199 @@ Attributes:
...
+#### Unsupported item
+
+1. Data synchronization between the master library and the slave library.
+2. Data inconsistency caused by the data synchronization delay of the master library and the slave library.
+3. Double writing or multiple writing in the main library.
+
+#### Code development example
+
+```java
+// 构建读写分离数据源, 读写分离数据源实现了DataSource接口, 可直接当做数据源处理. masterDataSource0, slaveDataSource00, slaveDataSource01等为使用DBCP等连接池配置的真实数据源
```
Comment (tristaZero) [inline docs/document/content/user-manual/shardingsphere-jdbc/configuration/java-api/change-history.en.md:206]: Hi, please translate them in English.

File: docs/document/content/user-manual/shardingsphere-jdbc/configuration/java-api/change-history.en.md [hunk 2]
```diff
@@ -31,4 +31,199 @@ Attributes:
...
+Map<String, DataSource> dataSourceMap = new HashMap<>();
+dataSourceMap.put("ms_0", masterSlaveDs0);
+dataSourceMap.put("ms_1", masterSlaveDs1);
+
+// 通过ShardingDataSourceFactory继续创建ShardingDataSource
+```
+
+
+
+
```
Comment (tristaZero) [inline docs/document/content/user-manual/shardingsphere-jdbc/configuration/java-api/change-history.en.md:228]: I guess these blank lines are not necessary, right? Please remove them.

**Review by tristaZero — CHANGES_REQUESTED** [review]
Hi @xiaoshen11 
Thanks for your PR, please look at my comments.

File: docs/document/content/user-manual/shardingsphere-jdbc/configuration/java-api/change-history.en.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title = "Change History"
 weight = 7
 +++
 
-## 5.0.0-alpha
+## ShardingSphere-5.0.0-alpha
```
Comment (yu199195) [inline docs/document/content/user-manual/shardingsphere-jdbc/configuration/java-api/change-history.en.md:6]: please remove it

File: docs/document/content/user-manual/shardingsphere-jdbc/configuration/java-api/change-history.cn.md [hunk 4]
```diff
@@ -31,4 +31,206 @@ weight = 7
...
+
+| *名称*                             | *数据类型* | *说明*                                                 |
+| :--------------------------------- | :--------- | :----------------------------------------------------- |
+| sql.show (?)                       | boolean    | 是否打印SQL解析和改写日志，默认值: false               |
+| executor.size (?)                  | int        | 用于SQL执行的工作线程数量，为零则表示无限制。默认值: 0 |
+| max.connections.size.per.query (?) | int        | 每个物理数据库为每次查询分配的最大连接数量。默认值: 1  |
+| check.table.metadata.enabled (?)   | boolean    | 是否在启动时检查分表元数据一致性，默认值: false        |
+
+
+
```
Comment (yu199195) [inline docs/document/content/user-manual/shardingsphere-jdbc/configuration/java-api/change-history.cn.md:75]: remove this line

File: docs/document/content/user-manual/shardingsphere-jdbc/configuration/java-api/change-history.en.md (hunk 1 above)
Comment (xiaoshen11) [reply docs/document/content/user-manual/shardingsphere-jdbc/configuration/java-api/change-history.en.md]: have translated

**Review by yu199195 — APPROVED** [review]

**Review by tristaZero — APPROVED** [review]

**Pull request merged**